packages/eslint-plugin/tests/rules/class-literal-property-style.test.ts

```diff
@@ -177,6 +177,24 @@ class Mx {
       `,
       options: ['getters'],
     },
+    {
+      code: `
+class Mx {
+  private readonly foo: string = 'bar';
```

Comment:
@bradzacher so change this test case to:

```
export class A {
  private readonly foo: string;

  constructor(foo: string) {
    this.foo = foo;
  }
}
```

Reply:
I would have both.

`private readonly foo: string = 'bar';` should be an `invalid` case for the config `options: ['getters']`, and it should be fixed appropriately.

As mentioned; the case with the assignment in the constructor should still be invalid, but there should be no auto-fix for it (use `output: null` to test this).